includes/ApplHandlingByStr.php

```diff
@@ -41,6 +82,44 @@ function showStatus( $stat_str )
 }
 
 
+function needAcceptor($status)
+{
+	$str = explode ( ';', $status );
+	for ( $i =  1;  $i < count ( $str )-1 ; $i++ )
+	{
+		$st = explode ( ',', $str[$i] )[1];
+		
+		if( $st == 'P' )
+		{
+			return false;
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->